lib/components/SearchResults.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export default ({results}) => {
+  if (results.length === 0) {
+    return <h1>No results yet!</h1>;
+  } else {
+    return (
+      <ul>{results.map((artist, track) =>
+        <li>
+          {artist}
+        </li>
+      )}</ul>
+    )
```

Comment:
if you replace lines 7-12 with `<div></div>` your double AJAX error should be resolved